src/metrics.js

```diff
@@ -264,19 +271,27 @@ function resetAll() {
   resetMetric(NAMESPACES.internalMetrics);
   resetMetric(NAMESPACES.endpoints);
   resetCustomMetrics();
+  resetCustomMetersMetrics();
 
 }
 
 function resetCustomMetrics() {
   for (var customMetricName in customMetrics) {
     if (customMetrics.hasOwnProperty(customMetricName)) {
-      let customNamespace = customMetricName.substring(0, customMetricName.indexOf("."))
+      let customNamespace = customMetricName.substring(0, customMetricName.indexOf("."));
       resetMetric(customNamespace);
     }
   }
   customMetrics = {};
 }
 
+function resetCustomMetersMetrics() {
```

Comment:
All of the other reset methods delete the metric so it will not appear in the next getMetrics request.